test/run

```diff
@@ -2,9 +2,7 @@
 # This is the expected entry point for Cockpit CI; will be called without
 # arguments but with an appropriate $TEST_OS, and optionally $TEST_SCENARIO
 
-export BACKEND="${BACKEND:-lorax-composer}"
-
-if [ "$BACKEND" == "osbuild-composer" ] || [ "$TEST_SCENARIO" == "osbuild-composer" ]; then
+if [ "$TEST_SCENARIO" == "osbuild-composer" ]; then
     rm -rf ./test/images/*
     export BACKEND="osbuild-composer"
     make BACKEND=osbuild-composer vm
```

Comment:
This shouldn't be necessary now that the backend is changing, no ?

Reply:
At some point I want to turn on the lorax scenario, so yes I think it needs to check since the lorax vm won't need osbuild-composer installed.

Reply:
Sure, the lorax tests don't need osbuild-composer but you have the BACKEND variable set to default to osbuild-composer in many places, Makefile for example, which will propagate to vm.install and it will install osbuild-composer anyway. 

The section in `test/run` which is guarded by the modified if condition was used to do a conditional make vm because the proper value of the backend variable didn't get propagated. But `vm.install` still installs osbuild-composer or lorax-composer. Once you merge these changes this will start failing for the lorax tests b/c there will be no lorax-composer available anymore.

Reply:
> Makefile for example, which will propagate to vm.install and it will install osbuild-composer anyway.

Ah, right! I guess for now it's not worth worrying about and I can revisit that when I get the lorax only tests ready to be run.

> Once you merge these changes this will start failing for the lorax tests b/c there will be no lorax-composer available anymore.

Which ones? The whole point of this PR is to remove them all. I'm only leaving the composer-cli tests running against osbuild-composer and eventually those will get moved elsewhere when composer-cli is rewritten.

Reply:
> Once you merge these changes this will start failing for the lorax tests b/c there will be no lorax-composer available anymore.

I'm wrong here. Ignore this.

Reply:
Having this extra section still doesn't make any sense b/c `make vm` will do exactly the same, b/c the BACKEND variable now defaults to osbuild-composer. So you always get osbuild-composer installed even if you don't use it. But this doesn't block merging of this PR and proceeding forward with the cleanup.

Reply:
Right, I'll revisit the vm setup when I'm ready to enable plain lorax testing. Thanks for the review!